### DISCUSSION TITLE
INVALID_IMAGE_OPTIMIZE_REQUEST sent when images are requested through proxy to Vercel

### DISCUSSION THREAD

**Discussion by Cha195** [vercel/next.js#67407] [Help]
### Summary

I have a Nextjs project that is deployed on Vercel. I have a domain that forwards requests to Vercel via Cloudfront. All the assets seem to return properly except images.
For example, https://perf-www.pocketpills.com/drug/ozempic sends the request to https://acq-web-sricharans-projects-2369f463.vercel.app/drug/ozempic
But the images don't show up in the custom domain while they work in the domain specified by Vercel. Any workaround to make next <Image /> tags work? Currently, I'm using the native img tags (in prod) which doesn't have any optimizations.

**Comment by MateusBrah** [comment]
I don't know if I understand your problem very well. but I believe it is related to the settings in the next.config.js.
![Captura de Tela 2024-07-03 às 16 55 29](https://github.com/vercel/next.js/assets/103467943/1074c22b-9b20-4d8c-890e-573c5452bb7f)
You must add the domain in the settings.
```

/** @type {import('next').NextConfig} */
const nextConfig = {
  images: {
    domains: ['domain-example.amazonaws.com'],
  },
}


module.exports = nextConfig

```

**Reply by Cha195** [reply]
This is already configured with the use of `remotePatterns` since `domains` is deprecated
